packages/core-flows/src/definition/cart/steps/update-tax-lines.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  CartLineItemDTO,
+  CartShippingMethodDTO,
+  CartWorkflowDTO,
+} from "@medusajs/types"
+import { StepResponse, createStep } from "@medusajs/workflows-sdk"
+import { updateTaxLinesWorkflow } from "../workflows"
+
+interface StepInput {
+  cart_or_cart_id: CartWorkflowDTO | string
+  items?: CartLineItemDTO[]
+  shipping_methods?: CartShippingMethodDTO[]
+}
+
+export const updateTaxLinesStepId = "update-tax-lines-step"
+export const updateTaxLinesStep = createStep(
```

Comment:
thought: this should potentially have a check to see if this is an automatic tax region.

Reply:
will bring this over to the next PR